config/manufacturer_specific.xml

```diff
@@ -1,1926 +1,1929 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why the huge change. Please fix up your editor from reformating the entire file and only add the entry that has been added/changed.

Reply:
Sorry, should be fixed now.  Wasn't sure on file revision to use since it was just updated.